lib/game.rb

```diff
@@ -0,0 +1,17 @@
+class Game
+  attr_reader :name, :cost, :national_drawing
+
+  def initialize(name, cost, national_drawing)
+    @name             = name
+    @cost             = cost
+    @national_drawing = true
+  end
+
+  def national_drawing?
+    if @national_drawing
+      true
+    else
+      false
+    end
+  end
+end
```

Comment:
Beacause `@national_drawing` is set to `true` above, this method will always return `true`.  But, if you set up the initialize per my comment, you can write the method like this:

```ruby
def national_drawing?
  @national_drawing
end
```